### ISSUE TITLE
WARNING  -  Config value 'plugins': Plugin 'mike' option 'alias_type': Unrecognised configuration name: alias_type

### ISSUE THREAD

**Issue by ThinksFast** [jimporter/mike#130]
Hello, I am trying to improve the SEO settings for a doc's implementation that is using mike.  My goal is to have the latest version of the docs viewable on a static URL path, like `/latest/*`.  I believe the correct way to do this is to use a config like this:

```
  - mike:
      alias_type: copy
      canonical_version: latest
```

However, both my fork and the parent project are getting this error during build (regardless of what `alias_type` is set to):

`WARNING  -  Config value 'plugins': Plugin 'mike' option 'alias_type': Unrecognised configuration name: alias_type`

And with the above config, nothing gets built in the `/latest/` directory of the repo.  Any suggestions on how to fix this?

**Details**
- Parent project == Starlite -> https://github.com/starlite-api/starlite
- My fork -> https://github.com/ThinksFast/starlite
- mike version == "1.1.2"
- mike config is here: https://github.com/ThinksFast/starlite/blob/64ca2ae254ad69c61a304e8d672f56f790e5f5f4/mkdocs.yml#L299
- The docs get built to a branch called `gh-pages`: https://github.com/ThinksFast/starlite/tree/gh-pages
- They are built using a Github Action. Screenshot attached showing the error: https://github.com/ThinksFast/starlite/actions/runs/3842119880/jobs/6543105208

![image](https://user-images.githubusercontent.com/114229148/210863613-bcbcf1f5-1269-4792-aa74-bc20b7dc6f62.png)

**Comment by jimporter** [comment]
The `alias_type` configuration option isn't available in the [v1.1.2 release](https://github.com/jimporter/mike/tree/v1.1.2). You'd need to use the latest development build for that.

`canonical_version` also only adds the appropriate HTML tag to existing docs. It doesn't add any new files. To have a `latest/` subdirectory, you'll need to specify `latest` as an alias via `mike deploy` or `mike alias`. This is covered in more detail in the README (see the link above).

**Comment by ThinksFast** [comment]
Ah, got it.  Thanks for the quick reply!  I made some changes to the deploy config and I think it's working the way I want it to.

And just to make sure I'm clear:  `alias_type: copy` does the same thing as specifying `latest` as an alias and adding `--no-redirect`, correct?

I'm using this as the deploy statement: 

`mike deploy $STARLITE_VERSION latest --push --force --update-aliases --no-redirect -t "$STARLITE_VERSION (latest)"`

**Comment by jimporter** [comment]
> And just to make sure I'm clear: `alias_type: copy` does the same thing as specifying `latest` as an alias and adding `--no-redirect`, correct?

Correct.

> I'm using this as the deploy statement:
> 
> `mike deploy $STARLITE_VERSION latest --push --force --update-aliases --no-redirect -t "$STARLITE_VERSION (latest)"`

I'd strongly recommend against using `--force`, for the same reason you generally wouldn't use `git push --force`. It's usually a sign that you're doing something risky, if not outright wrong.

**Comment by jimporter** [comment]
Closing this, as there have been no further comments in the past week. If you're still having issues though, feel free to reply back and I can reopen this.

**Comment by athshean** [comment]
> The `alias_type` configuration option isn't available in the [v1.1.2 release](https://github.com/jimporter/mike/tree/v1.1.2). You'd need to use the latest development build for that.
> 
> `canonical_version` also only adds the appropriate HTML tag to existing docs. It doesn't add any new files. To have a `latest/` subdirectory, you'll need to specify `latest` as an alias via `mike deploy` or `mike alias`. This is covered in more detail in the README (see the link above).

How to use the latest development build? The docs don't say where this build can be installed from.

**Comment by jimporter** [comment]
> How to use the latest development build? The docs don't say where this build can be installed from.

From the repository: https://docs.readthedocs.io/en/stable/guides/private-python-packages.html#from-a-git-repository
